docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2525,3 +2525,11 @@ Move an app plugin (referenced by its id), including all its pages, to a specifi
 
 Move an individual app plugin page (referenced by its `path` field) to a specific navigation section.
 Format: `<pageUrl> = <sectionId> <sortWeight>`
+
+## [public_dashboards]
+
+This section configures the [public dashboards]({{< relref "../../dashboards/dashboard-public" >}}) feature.
+
+### enabled
+
+Set this to `false` to disable the public dashboards feature. This will prevent users from creating new public dashboards and disable all existing ones.
```

Comment:
```suggestion
Set this to `false` to disable the public dashboards feature. This prevents users from creating new public dashboards and disabling existing ones.
```

Reply:
This doesn't have the same meaning but I can change it to something like:
```suggestion
Set this to `false` to disable the public dashboards feature. This prevents users from creating new public dashboards and disables existing ones.
```

Reply:
good catch, I missed needing a change to 'disables'. Thank you.